composer.json

```
@@ -11,27 +11,31 @@
         }
     ],
     "require": {
-        "php": "^7.0 || ^8.0",
-        "psr/log": "^1.0 || ^2 || ^3",
-        "php-http/client-common": "^1.9 || ^2.0",
-        "php-http/message": "^1.0",
-        "symfony/polyfill-php73": "^1.17"
-    },
-    "require-dev": {
-        "phpspec/phpspec": "^5.1 || ^6.0"
+        "php": "^8.2",
+        "psr/log": "^2 || ^3",
+        "php-http/client-common": "^2.0",
+        "php-http/message": "^1.13"
     },
     "autoload": {
         "psr-4": {
             "Http\\Client\\Common\\Plugin\\": "src/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "Http\\Client\\Common\\Plugin\\": "tests/"
+        }
+    },
     "scripts": {
-        "test": "vendor/bin/phpspec run",
-        "test-ci": "vendor/bin/phpspec run -c phpspec.yml.ci"
+        "test": "vendor/bin/phpunit"
     },
     "extra": {
         "branch-alias": {
             "dev-master": "1.2-dev"
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^11",
```

lets use this to fix lowest builds:

```suggestion
        "phpunit/phpunit": "^11.4.3",
```

(as its a dev dependency i don't care if there would be some version between 11.0.0 and this one that would also work.)

---

Good idea!